content/post/2017-04-29-introduction-of-liechi-s-site.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Introduction of Liechi's site
+author: Liechi Zhang
+date: '2017-04-29'
+slug: introduction-of-liechi-s-site
```

Comment:
I recommend a shorter slug, e.g. `liechi-site`

You also need a `description` in YAML; see the preview: https://deploy-preview-9--rbind-support.netlify.com